File: experimental/lupa_ab/test_core.lua
local ab = require 'core'
local JSON = require "JSON"


ab_struct = ab.init_ab("My_Config")
local ab_tbl = {}
ab_tbl['factor'] = 2
out_json = ab.sum_ab(ab_struct, JSON:encode(ab_tbl))
out_tbl = JSON:decode(out_json)
for i, v in pairs(out_tbl) do
  print(i, v)
end
ab.print_ab(ab_struct)
ab.free_ab(ab_struct)
print("=============================================")

File: experimental/lupa_ab/test_core.py
import core as ab
import json


ab_struct = ab.init_ab("my_config")
sum_result = ab.sum_ab(ab_struct, factor=2)
sum_dict = json.loads(sum_result)
for i, v in sum_dict.items():
    print(i, v)
ab.print_ab(ab_struct)
ab.free_ab(ab_struct)
